lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -479,32 +425,33 @@ UsdTransform3dMayaXformStack::rotateCmd(double x, double y, double z)
     GfVec3f           v(x, y, z);
     CvtRotXYZToAttrFn cvt = hasRotate ? getCvtRotXYZToAttrFn(op.GetOpName()) : toXYZ;
 
```

Comment:
For the changes below, I suggest you "Hide whitespace" in the diff preferences, under the cog at the top of the review page.